vtexdocs/devportal
# Fix/search input results

#### What is the purpose of this pull request?
Fix:
- `legacyBehavior` warning on search results
- When the user clicks on the search results, close the results box
- Change the feedback message when the search returns nothing
- Guides page icons (missing)

#### How should this be manually tested?
Go to [this page](https://deploy-preview-143--elated-hoover-5c29bf.netlify.app/) and check if everything works fine.

#### Types of changes
